src/odemis/acq/fastem.py

```diff
@@ -318,6 +318,9 @@ def run(self):
             # Remove references to the megafield once the acquisition is finished/cancelled.
             self._fields_remaining.clear()
 
+            # FIXME auto blanking not working properly, so force beam blanking after image acquisition for now.
```

Comment:
Autoblanking will never work in external mode as far as I know.

Reply:
Indeed, auto-blanking is not going to work with external mode. Actually, Kornee has some new wrapper component almost ready, for the ENZEL, to add a "None" option to the blanker VA in such case, but that relies on the detector behaving "normally" (ie, not the extended API of the ASM DataFlow). So don't count on it anytime soon. It's simpler to just handle it explicitly here.

Reply:
Why will autoblanking never work in external mode? I can get a value for whether the beam is blanked or not, and I can blank the beam in external mode.

Reply:
Because autoblanking means "unblank" when the DataFlow linked to this emitter has subscribers. However, as the .blanker VA is on the xt_client driver and the DataFlow is on the ASM, there is currently no way to connect one to the other.
The approach taken by Kornee to handle this is to have a wrapper for the DataFlow, which takes care of switching the .blanker VA. With some extension, it could work with the special FASTEM/ASM DataFlow API, but as all this is so ad-hoc, I don't see much the point compared to just explicitly controlling the blanker in the acquisition code, as you do.